tokens.json

```diff
@@ -0,0 +1,3376 @@
+[
```

Comment:
We definitely thought so! Especially since relying on relayer APIs to get the correct address/symbol to token information mapping tends to yield different information